src/ray/protobuf/ray_client.proto

```diff
@@ -114,6 +114,8 @@ message PutRequest {
   int32 total_chunks = 4;
   // Total size in bytes of the data being put
   int64 total_size = 5;
+  // The owner of the put
+  bytes _owner_id = 6;
```

Comment:
nit: any reason why this needs to be prefixed with underscore?

Reply:
I'm just following the non-client mode API:

https://github.com/ray-project/ray/blob/master/python/ray/_private/worker.py#L2314

and it's ray.put(_owner=xxx). But this is protobuf, I think I can change it to the one without `_`. Up to you.

Reply:
If you can drop from the protobuf that would be nice, I think there's some [wonky behavior](https://github.com/golang/protobuf/issues/201) sometimes when you prefix with non-alphabetical characters.